### PR TITLE
Episode 8: Corrections for 0:45–1:00

### PR DESCRIPTION
**Pull request by pootsbook**
Only two points of obscurity:

544: I've inserted 'fake it' as it conveys the point,
however I believe Camille uses a French expression resembling
'sens amical' in reference to Sam’s 'adversarial'.

645: Unsure of what 'concom' means in this context

### CODE REVIEW

**Comment by nelstrom** [comment]
@pootsbook thanks for these corrections! I'll review and merge later...

**Comment by nelstrom** [comment]
> 544: I've inserted 'fake it' as it conveys the point,
> however I believe Camille uses a French expression resembling
> 'sens amical' in reference to Sam’s 'adversarial'.

That seems like a good guess. Maybe it's 'sans amical', which would mean 'not friendly'... 

Maybe @camilleldn can listen to [this 15s long excerpt](http://media.vimcasts.org/videos/tmp/camille-says-something-in-french.mov) and let us know what she actually said?

File: 8/sam-phippen-captions.srt [hunk 1]
```diff
@@ -2290,12 +2290,13 @@ true for any method on this object.
...
 00:45:06,976 --> 00:45:13,152
-It should also inaud, but we'll worry about that later.
+It should also handle method_missing correctly,
+but we'll worry about that later.
 
 496
 00:45:14,890 --> 00:45:18,784
 And this might be a good place to
-describe the response to method like this
+describe the respond_to? method like this
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:2299]: :+1:

File: 8/sam-phippen-captions.srt [hunk 2]
```diff
@@ -2318,26 +2319,27 @@ argument which is the method name,
...
 00:45:47,664 --> 00:45:54,192
-So because our object responds
-to all methods, it's going to return true.
+So because our object responds to all methods,
+it's going to need to return true.
 
 504
 00:45:54,190 --> 00:46:01,744
-So expect subject.respond
+So expect(subject.respond...
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:2337]: This hanging paren is going to mess with people's heads. :trollface:

File: 8/sam-phippen-captions.srt [hunk 3]
```diff
@@ -2462,25 +2465,26 @@ SP: Yes, so we can dot chain.
...
 534
 00:48:34,480 --> 00:48:40,736
-It's probably worth noting here, that the reason I'm using B here instead of inaud
+It's probably worth noting here, that the
+reason I'm using 'be' here instead of 'eq'
 
 535
 00:48:41,040 --> 00:48:46,480
-is that B does exact object comparison
+is that 'be' does exact object comparison
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:2473]: :+1:

File: 8/sam-phippen-captions.srt [hunk 4]
```diff
@@ -2605,17 +2609,17 @@ At this point I expect both of these tests to fail.
 565
 00:51:13,960 --> 00:51:21,008
 And they do, with false and
-undefined method 916 etc. 4 bond spy.
+undefined method 916 etc. for #<Bond::Spy>.
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:2612]: :+1:

File: 8/sam-phippen-captions.srt [hunk 5]
```diff
@@ -2725,12 +2732,12 @@ test and check where we're at.
...
 00:53:33,920 --> 00:53:36,896
 We still haven't done this
-bond did you receive method.
+bond_did_you_receive? method.
 
 592
 00:53:36,890 --> 00:53:47,024
 So if we run that we can see...
-expect to receive method name.
+Expected to receive :method name.
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:2740]: I think this should be: `Expected to receive :method_name`

File: 8/sam-phippen-captions.srt [hunk 6]
```diff
@@ -2932,11 +2939,11 @@ CB: Why are you hesitating on this one?
 636
 00:57:37,040 --> 00:57:43,088
 SP: I'm trying to remember if this
-wants to be single 'at' or a double 'at'.
+wants to be single '@' or a double '@@'.
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:2942]: Should be:

> single '@' or a double '@'

File: 8/sam-phippen-captions.srt [hunk 7]
```diff
@@ -2971,13 +2979,13 @@ Actually, the other thing we could do...
 
 645
 00:58:26,640 --> 00:58:39,360
-is just have a... this... and then we can
-reference bond concom proxies and do things to it
+is just have... this... and then we can
+reference Bond concom proxies and do things to it
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:2983]: I hear "colon-colon":

> we can reference Bond::Proxies

Comment (pootsbook) [inline 8/sam-phippen-captions.srt:2983]: That is a great catch, I had no idea what was going on there, even after a couple of listens. For fidelity, we can then change it to reflect the constant, i.e. `Bond::PROXIES`

File: 8/sam-phippen-captions.srt (hunk 5 above)
Comment (pootsbook) [inline 8/sam-phippen-captions.srt:2740]: @nelstrom Would you like me to make these modifications and push to the PR?

**Comment by nelstrom** [comment]
@pootsbook I've already merged this PR and made those alterations in the target branch. Thanks again for your help.

File: 8/sam-phippen-captions.srt [hunk 8]
```diff
@@ -2507,12 +2511,12 @@ So I'm actually just going to write the method.
 
 544
 00:49:23,960 --> 00:49:27,952
-CB: Oh of course, inaud to make it pass.
+CB: Oh of course, fake it to make it pass.
```
Comment (nelstrom) [inline 8/sam-phippen-captions.srt:2514]: I've revised this: https://github.com/fromPeerToPeer/captions/commit/76377ccb74947b46cf7ec389af90a4574d3e1537

**Pull request merged**